docs/gettingstarted/blender.rst

```diff
@@ -69,12 +72,53 @@ Now configure Blender to use the newly installed environment or any environment
 .. code-block:: bash
 
     conda activate blender
-    python -m compas_blender.install /Applications/blender.app/Contents/Resources/2.83
+    python -m compas_blender.install /Applications/blender.app/Contents/Resources/2.91
 
-Note that the path ``/Applications/blender.app/Contents/Resources/2.83`` might be different
+Note that the path ``/Applications/blender.app/Contents/Resources/2.91`` might be different
 if you have another version of Blender intalled.
 Check your version of Blender and change the path accordingly.
 
+.. raw:: html
+
+    </div>
+    <div class="tab-pane" id="replace_python_linux">
+
+If you wish to install a new python environment with COMPAS, open the Terminal and type the following:
+
+.. code-block:: bash
+
+    conda config --add channels conda-forge
+    conda create -n blender python=3.7 COMPAS
+
+Now configure Blender to use the newly installed environment or any environment in which you have COMPAS.
+
+First, backup the original python:
+
+.. code-block:: bash
+
+    mv <blender_folder/2.81/python/> <blender_folder/2.81/original_python>
+
+Activate the conda environment and find the location of the python interpreter:
+
+.. code-block:: bash
+
+    conda activate blender
+    which python
+
+create a symbolic link between the folders:
+
+.. code-block:: bash
+
+    ln -s <env folder> <blender_folder/2.91/>
+
+rename the folder so that blender can find it:
+
+.. code-block:: bash
+
+    ln -s <env folder> <blender_folder/2.91/>
```

Comment:
why not use `compas_blender.install` for this?

Reply:
did not work on my machine. I could look into that. this is just a quick fix.